pkg/apis/core/validation/validation.go

```diff
@@ -5253,12 +5252,6 @@ func ValidateSecurityContext(sc *core.SecurityContext, fldPath *field.Path) fiel
 		return allErrs
 	}
 
-	if sc.Privileged != nil {
```

Comment:
Oh... pull on a thread and the whole universe unravels. It turns out the capabilities feature is a thread that also runs through the API server, where there is another `--allow-privileged` flag that never got marked deprecated... which means we can't remove this piece yet.

I was hoping we could rip the AllowPrivileged part of the capabilities feature out altogether, but it looks like we have to hang on to it internally until we can deprecate remove the flag from the API server as well. The API server piece should probably be discussed in sig-auth @tallclair.

@charrywanganthony let's back this PR up to just removing the flag from Kubelet. I'm very sorry for making you repeat this.... :sob:

Reply:
Sorry for delay, as you requested i update the commit. PTAL